src/algorithms/peps_opt.jl

```diff
@@ -49,7 +51,7 @@ Different levels of output verbosity can be activated using `verbosity` (0, 1 or
     )
     reuse_env::Bool = true  # Reuse environment of previous optimization as initial guess for next
     gradient_alg::G = GeomSum() # Algorithm to solve gradient linear problem
-    verbosity::Int = 0
+    verbosity::Int = 1 # currently unused?
```

Comment:
Yes, this is currently unused and I think that by now it actually is redundant. All of the printing information happens either in the `optimizer` or the `gradient_alg` - both of which have their own verbosity. So how about we get rid of it?